x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/actions_menu.tsx

```diff
@@ -171,6 +174,24 @@ export const AgentPolicyActionMenu = memo<{
                 </AgentPolicyDeleteProvider>,
               ];
 
+          if (!agentPolicy?.is_managed) {
```

Comment:
I have a doubt whether this check is sufficient, especially regarding `authz.fleet`.
@nchaulet Could you comment on this?

Reply:
This probably be accessible to user with `authz.fleet.allAgents` as it's the permission needed on the API side